### PR TITLE
Adjust swift template for in project generation

### PR DESCRIPTION
**Pull request by telip007**

### CODE REVIEW

File: Templates/Swift/template.yml [hunk 1]
```diff
@@ -20,38 +20,33 @@ options:
     ID: String
     DateTime: Date
     File: Data
-  dependencies:
-    - name: Alamofire
-      pod: Alamofire
-      github: Alamofire/Alamofire
-      version: 4.9.0
-    - name: Poly
```
Comment (EmDee) [inline Templates/Swift/template.yml:28]: I saw that we have added the `import Poly` manually in the model files. We should define the dependencies here and retrieve the values in the model files.

Comment (EmDee) [reply Templates/Swift/template.yml:28]: @telip007 Ping

Comment (telip007) [reply Templates/Swift/template.yml:28]: I already removed the `import Poly` from the models since we include them as source code.

Comment (telip007) [reply Templates/Swift/template.yml:28]: Added changes.

**Pull request merged**